archetypes/business-service-spring-boot-starter/src/main/resources/archetype-resources/pom.xml

```diff
@@ -52,8 +52,9 @@
       <artifactId>spring-boot-starter-provider</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.hibernate</groupId>
+      <groupId>org.hibernate.validator</groupId>
       <artifactId>hibernate-validator</artifactId>
+      <version>6.0.2.Final</version>
```

Comment:
Maybe we should add this to dependency mangement, not specify version here.

Reply:
i will submit a new PR to fix this.